### DISCUSSION TITLE
Play from file system with Node.js, instead of a remote src URL

### DISCUSSION THREAD

**Discussion by bthj** [lhz516/react-h5-audio-player#229] [Q&A]
Is it possible to use this player to play local files, e.g. in a desktop app with access to files on disk via the Node.js API ?

**Comment by lhz516** [comment]
You mean using it in something like Electron? Not pretty sure, you can test it out

**Reply by bthj** [reply]
That was the idea. Will report back if I try it out.

**Reply by apetersonflc** [reply]
> That was the idea. Will report back if I try it out.

Any luck?

**Comment by aspyk** [comment]
I made some tests on the same issue and it seems converting audio file to base64 could solve it...  Something like:
```
const fileData = fs.readFileSync(absolutePath);
const base64 = `data:audio/mp3;base64,${fileData.toString("base64")}`;
```
With this my simple packaged electron app managed to play audio file from other directories. I still have to double check to avoid false positive though :)
